greenheart/simulation/technologies/ammonia/ammonia.py

```diff
@@ -0,0 +1,694 @@
+        cost=costs.capex_boiler,
+        depr_type="MACRS",
+        depr_period=7,
+        refurb=[0],
+    )
+    pf.add_capital_item(
+        name="Cooling Tower",
+        cost=costs.capex_cooling_tower,
+        depr_type="MACRS",
+        depr_period=7,
+        refurb=[0],
+    )
+    pf.add_capital_item(
+        name="Depreciable Nonequipment",
+        cost=costs.capex_depreciable_nonequipment,
+        depr_type="MACRS",
+        depr_period=7,
+        refurb=[0],
+    )
+
+    # -------------------------------------- Add fixed costs--------------------------------
+    pf.add_fixed_cost(
+        name="Labor Cost",
+        usage=1,
+        unit="$/year",
+        cost=costs.labor_cost,
+        escalation=gen_inflation,
+    )
+    pf.add_fixed_cost(
+        name="Maintenance Cost",
+        usage=1,
+        unit="$/year",
+        cost=costs.maintenance_cost,
+        escalation=gen_inflation,
+    )
+    pf.add_fixed_cost(
+        name="Administrative Expense",
+        usage=1,
+        unit="$/year",
+        cost=costs.general_administration_cost,
+        escalation=gen_inflation,
+    )
+    pf.add_fixed_cost(
+        name="Property tax and insurance",
+        usage=1,
+        unit="$/year",
+        cost=costs.property_tax_insurance,
+        escalation=0.0,
+    )
+
+    # ---------------------- Add feedstocks, note the various cost options-------------------
+    pf.add_feedstock(
+        name="Hydrogen",
+        usage=feedstocks.hydrogen_consumption,
+        unit="kilogram of hydrogen per kilogram of ammonia",
+        cost=feedstocks.hydrogen_cost,
+        escalation=gen_inflation,
+    )
+
+    pf.add_feedstock(
+        name="Electricity",
+        usage=feedstocks.electricity_consumption,
+        unit="MWh per kilogram of ammonia",
+        cost=config.grid_prices,
+        escalation=gen_inflation,
+    )
+    pf.add_feedstock(
+        name="Cooling water",
+        usage=feedstocks.cooling_water_consumption,
+        unit="Gallon per kilogram of ammonia",
+        cost=feedstocks.cooling_water_cost,
+        escalation=gen_inflation,
+    )
+    pf.add_feedstock(
+        name="Iron based catalyst",
+        usage=feedstocks.iron_based_catalyst_consumption,
+        unit="kilogram of catalyst per kilogram of ammonia",
+        cost=feedstocks.iron_based_catalyst_cost,
+        escalation=gen_inflation,
+    )
+    pf.add_coproduct(
+        name="Oxygen byproduct",
+        usage=feedstocks.oxygen_byproduct,
+        unit="kilogram of oxygen per kilogram of ammonia",
+        cost=feedstocks.oxygen_cost,
+        escalation=gen_inflation,
+    )
+
+    # ------------------------------ Sovle for breakeven price ---------------------------
+
+    sol = pf.solve_price()
+
+    summary = pf.get_summary_vals()
+
+    price_breakdown = pf.get_cost_breakdown()
+
+    price_breakdown_air_separation_by_cryogenic = price_breakdown.loc[
+        price_breakdown["Name"] == "Air Separation by Cryogenic", "NPV"
```

Comment:
The price breakdown code seems more complicated than necessary. Why do we need to pull out each item individually?

Reply:
I agree, I was wondering about this, I left it as it was so we could decide if we needed it for some reason